<ui version="4.0" >
 <class>ToolbarDialog</class>
 <widget class="QDialog" name="ToolbarDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>796</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize" >
   <size>
    <width>796</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize" >
   <size>
    <width>796</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle" >
   <string>Configure Toolbars</string>
  </property>
  <widget class="QPushButton" name="buttonDone" >
   <property name="geometry" >
    <rect>
     <x>570</x>
     <y>550</y>
     <width>100</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text" >
    <string>OK</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboButtonStyle" >
   <property name="geometry" >
    <rect>
     <x>120</x>
     <y>440</y>
     <width>661</width>
     <height>31</height>
    </rect>
   </property>
   <item>
    <property name="text" >
     <string>Icon Only</string>
    </property>
   </item>
   <item>
    <property name="text" >
     <string>Text Only</string>
    </property>
   </item>
   <item>
    <property name="text" >
     <string>Text beside Icon</string>
    </property>
   </item>
   <item>
    <property name="text" >
     <string>Text below Icon</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="comboIconSize" >
   <property name="geometry" >
    <rect>
     <x>120</x>
     <y>480</y>
     <width>661</width>
     <height>31</height>
    </rect>
   </property>
   <item>
    <property name="text" >
     <string>16x16</string>
    </property>
   </item>
   <item>
    <property name="text" >
     <string>32x32</string>
    </property>
   </item>
   <item>
    <property name="text" >
     <string>48x48</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>440</y>
     <width>100</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text" >
    <string>Text Position:</string>
   </property>
   <property name="alignment" >
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_3" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>480</y>
     <width>100</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text" >
    <string>Icon Size:</string>
   </property>
   <property name="alignment" >
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_4" >
   <property name="geometry" >
    <rect>
     <x>480</x>
     <y>80</y>
     <width>122</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text" >
    <string>Current Actions:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>10</y>
     <width>58</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text" >
    <string>ToolBar:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonCancel" >
   <property name="geometry" >
    <rect>
     <x>680</x>
     <y>550</y>
     <width>100</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text" >
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonDefault" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>550</y>
     <width>100</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text" >
    <string>Default</string>
   </property>
  </widget>
  <widget class="Line" name="line" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>520</y>
     <width>791</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation" >
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonRemove" >
   <property name="geometry" >
    <rect>
     <x>339</x>
     <y>275</y>
     <width>30</width>
     <height>29</height>
    </rect>
   </property>
   <property name="sizePolicy" >
    <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="maximumSize" >
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
   <property name="text" >
    <string>&lt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonUp" >
   <property name="geometry" >
    <rect>
     <x>380</x>
     <y>240</y>
     <width>30</width>
     <height>29</height>
    </rect>
   </property>
   <property name="sizePolicy" >
    <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="maximumSize" >
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
   <property name="text" >
    <string>^</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonDown" >
   <property name="geometry" >
    <rect>
     <x>380</x>
     <y>310</y>
     <width>30</width>
     <height>29</height>
    </rect>
   </property>
   <property name="sizePolicy" >
    <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="maximumSize" >
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
   <property name="baseSize" >
    <size>
     <width>77</width>
     <height>27</height>
    </size>
   </property>
   <property name="text" >
    <string>V</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonAdd" >
   <property name="geometry" >
    <rect>
     <x>421</x>
     <y>275</y>
     <width>30</width>
     <height>29</height>
    </rect>
   </property>
   <property name="sizePolicy" >
    <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="maximumSize" >
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
   <property name="baseSize" >
    <size>
     <width>41</width>
     <height>27</height>
    </size>
   </property>
   <property name="text" >
    <string>></string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboToolbars" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>40</y>
     <width>771</width>
     <height>32</height>
    </rect>
   </property>
  </widget>
  <widget class="QListWidget" name="listUsedActions" >
   <property name="geometry" >
    <rect>
     <x>480</x>
     <y>120</y>
     <width>301</width>
     <height>301</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>80</y>
     <width>132</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text" >
    <string>Available Actions:</string>
   </property>
  </widget>
  <widget class="QListWidget" name="listAllActions" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>119</y>
     <width>311</width>
     <height>301</height>
    </rect>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>comboToolbars</tabstop>
  <tabstop>listAllActions</tabstop>
  <tabstop>listUsedActions</tabstop>
  <tabstop>buttonRemove</tabstop>
  <tabstop>buttonAdd</tabstop>
  <tabstop>buttonUp</tabstop>
  <tabstop>buttonDown</tabstop>
  <tabstop>comboButtonStyle</tabstop>
  <tabstop>comboIconSize</tabstop>
  <tabstop>buttonDone</tabstop>
  <tabstop>buttonCancel</tabstop>
  <tabstop>buttonDefault</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonDone</sender>
   <signal>clicked()</signal>
   <receiver>ToolbarDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>278</x>
     <y>253</y>
    </hint>
    <hint type="destinationlabel" >
     <x>96</x>
     <y>254</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonCancel</sender>
   <signal>clicked()</signal>
   <receiver>ToolbarDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>742</x>
     <y>562</y>
    </hint>
    <hint type="destinationlabel" >
     <x>309</x>
     <y>555</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
